perceval/backends/core/mbox.py

```diff
@@ -18,9 +18,14 @@
 # Authors:
 #     Santiago Dueñas <sduenas@bitergia.com>
 #     Germán Poo-Caamaño <gpoo@gnome.org>
+#     Stephan Barth <stephan.barth@gmail.com>
+#     valerio <valerio.cosentino@gmail.com>
+#     Valerio Cosentino <valcos@bitergia.com>
```

Comment:
Hi @valeriocos 

Just a small doubt, do these both belong to you? :sweat_smile: 
If yes, do I need to merge them or leave it like this?

Reply:
eheh thanks! yes, it belongs to me, can you merge them?